samples/tfm/tfm_hello_world/README.rst

```diff
@@ -14,9 +14,8 @@ Requirements
 
 The sample supports the following development kits:
 
-.. table-from-rows:: /includes/sample_board_rows.txt
+.. table-from-sample-yaml:: /includes/sample_board_rows.txt
```

Comment:
Can I do this change immediately to https://github.com/nrfconnect/sdk-nrf/pull/7148/files#diff-e40ce80d37ad512bc8ce14a370eb1fa2177b03d70e9d0903b1815174342974afR17
or do I need to wait for this PR to be merged?